CONTRIBUTING.md

```diff
@@ -109,17 +109,15 @@ We want to improve the quality of the open-source vulnerability assessment tool
 
 Analyzing issue reports can be a lot of effort. Any help is welcome! Go to [the Github issue tracker](https://github.com/SAP/open-source-vulnerability-mgmt/issues?state=open) and find an open issue which needs additional work or a bugfix.
 
-Additional work may be further information, or a minimized jsbin example or gist, or it might be a hint that helps understanding the issue. Maybe you can even find and [contribute](#contribute-code) a bugfix?
```

Comment:
Because it is specific only to UI5, It suggest to use https://jsbin.com/?html,output in order to create a debuggable example. Since we do not expose any JS library it does not make any sense to suggest users to use JsBin.

Reply:
jsbin could be used for frontend-* but i agree with you, that's not the most crucial part of the project :) 

Anyways they are just examples, and gist is the another alternative.  In any case i would keep the item and remove the examples if you think they are misleading